src/rulesets/oas/index.json

```diff
@@ -115,7 +115,12 @@
       "description": "Example should have either a `value` or `externalValue` field.",
       "recommended": true,
       "type": "style",
-      "given": "$..examples.*",
+      "given": [
+        "$..content.*.examples.*",
```

Comment:
Any chance we could narrow these down?
`$..` path expressions are fairly expensive, as we need to traverse deeply pretty much the whole object.
If we could have something more or less close to
`$.paths.*..parameters` and `$.components..parameters` that would be already a huge step forward.

Same for `content` and `headers`. If it happens to be very troublesome to implement, I can live with it.

Reply:
It's just a matter of going through the spec and compiling all cases. I'd expect that it blows up the list quite a lot . Also, I'd not be sure whether I found all cases and I would expect anybody to carefully double-check it. So I'll only do the work if anybody would agree on checking it.

Reply:
I just tried to resolve `"$..content.*.examples.*",` and it seems to become 
```
        "$.paths.*.*.requestBody.content.*.examples.*",
	"$.paths.*.*.callbacks.*.*.requestBody.content.*.examples.*",
        "$.components.callbacks.*.*.*.requestBody.content.*.examples.*",
        "$.components.requestBodies.*.content.*.examples.*",
```
BUT: I'm not at all sure whether that's correct, but I tried my best. And I think that was the easiest case. Also, some of the * could be replaced by (get or post or patch or put or ...), but I'm not sure how OR works in JSON Path without blowing the expression to be overly long and complicated. So I think I'd not be able to provide a proper list at the moment, time is too limited.

Reply:
I committed the version above, the tests succeed.

Reply:
@m-mohr yeah, that's much better. Thanks!